app/assets/stylesheets/components/home_page_header.scss

```diff
@@ -1,25 +1,24 @@
-header {
-    padding-bottom: 60px;
+.hero {
+  text-align: center;
+  width: 100%;
 
-  h1 {
-    margin-bottom: 30px;
+  header {
+    padding-bottom: 3.75rem;
```

Comment:
ah.. that post mentions proportions.. using REMS with integers means everything ends up being divisible by 16.. which can be useful if you don't have a pixel-perfect design you're trying to emulate.. makes it all nice and even.  But here we're trying to be exact and that leads us to doing things like 3.75REMS which might as well be 30px.

I might be missing something though lol, and I still don't _really_ care.